### DISCUSSION TITLE
card locked after click

### DISCUSSION THREAD

**Discussion by AndreasSchmidt87** [neuefische/questions#213] [Web] [answered]
Hello frinds,

the cards in my memory game are supposed to stay face up, but they always turn back. How do I get them to remain uncovered after the click? Here is my code:

![Bildschirmfoto 2022-10-05 um 13 05 51](https://user-images.githubusercontent.com/76123735/194046545-afd92115-16a6-434b-8a49-5d12a03bc11d.png)

![Bildschirmfoto 2022-10-05 um 13 06 04](https://user-images.githubusercontent.com/76123735/194046563-6cf18e67-8a4e-4b04-be2b-6a3e580f53e9.png)

can someone help me, thanks in advance.

**Comment by NikRoe** [comment]
Hey Andi,

please send us a link to a Pull Request.

Thanks in advance :)

**Comment by ThomasKirst** [comment, marked as answer]
`&:active` –> `&.active`

**Reply by ThomasKirst** [reply]
I also left a comment in your PR ;)

You want to react to the css class `.active` and not to the css pseudo class `:active`.
